src/style.css

```
@@ -1,3 +1,86 @@
+:root {
+  --color-primary: #00acdc;
+  /* nav__link settings without margin */
+  --nav__link__padding: 22.5px 0;
+  --nav__link__font-family: roboto, sans-serif;
+  --nav__link__font-size: 12px;
+  --nav__link__font-weight: 500;
+  --nav__link__text-decoration: none;
+  --nav__link__color: black;
+  --nav__link__text-transform: uppercase;
+  --nav__link__display: block;
+  /* nav__link settings without margin */
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  display: flex;
+  margin: 0 50px;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__link {
+  margin: 0 0 0 20px;
+  padding: var(--nav__link__padding);
+  font-family: var(--nav__link__font-family);
+  font-size: var(--nav__link__font-size);
+  font-weight: var(--nav__link__font-weight);
+  text-decoration: var(--nav__link__text-decoration);
+  color: var(--nav__link__color);
+  text-transform: var(--nav__link__text-transform);
+  display: var(--nav__link__display);
+}
+
+.nav__link_no_margin {
+  padding: var(--nav__link__padding);
+  font-family: var(--nav__link__font-family);
+  font-size: var(--nav__link__font-size);
+  font-weight: var(--nav__link__font-weight);
+  text-decoration: var(--nav__link__text-decoration);
+  color: var(--nav__link__color);
+  text-transform: var(--nav__link__text-transform);
+  display: var(--nav__link__display);
+}
```

nav__link_no_margin is not correct name
 - no_margin should be a modifacator, for instance nav__link--no-margins 
 - don't need to duplicated all styles from nav__link
 - modificator -`-no-margins` should reset only margins 
```suggestion
.nav__link--no-margin {
  margin: 0;
}
```

please check BEM documentation 
https://en.bem.info/methodology/quick-start/